acceptance/src/data.rs

```diff
@@ -127,9 +128,9 @@ pub async fn assert_scan_data(engine: Arc<dyn Engine>, test_case: &TestCaseInfo)
     let mut schema = None;
     let batches: Vec<RecordBatch> = scan
         .execute(engine)?
-        .into_iter()
-        .map(|res| {
-            let data = res.raw_data.unwrap();
+        .map(|sr| -> DeltaResult<_> {
+            let sr = sr?;
+            let data = sr.raw_data?;
```

Comment:
naming nit for clarity?
```suggestion
        .map(|scan_result| -> DeltaResult<_> {
            let scan_result = scan_result?;
            let data = scan_result.raw_data?;
```